utils/bluld/files/bluld.init

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2020 Keith T. Garner
```

Comment:
We have 2021. ;)

Reply:
but I haven't touched the file in a year :) lol